api/src/paths/project/{projectId}/survey/{surveyId}/sample-site/{surveySampleSiteId}/sample-method/index.ts

```diff
@@ -94,7 +94,7 @@ GET.apiDoc = {
                     survey_sample_site_id: {
                       type: 'integer'
                     },
-                    method_lookup_id: {
+                    method_technique_id: {
                       type: 'integer'
```

Comment:
min

Reply:
Updated, added some other missing fields to the spec as well.